Source/Demos/Demo.Particles/Game1.cs

```diff
@@ -93,7 +93,7 @@ private void ParticleInit(TextureRegion2D textureRegion)
             {
                 Emitters = new[]
                 {
-                    new ParticleEmitter(500, TimeSpan.FromSeconds(2.5), Profile.Ring(150f, Profile.CircleRadiation.In))
+                    new ParticleEmitter(500, TimeSpan.FromSeconds(2.5), Profile.Ring(new CircleF(Vector2.Zero, 150f), Profile.CircleRadiation.In))
```

Comment:
Oh it's definitely not, I just wanted to preserve the way the demo worked for now. Should've made the circle in the ring method, my bad.